src/Raven.Studio/typescript/viewmodels/database/indexes/indexPerformance.ts

```diff
@@ -1152,10 +1152,22 @@ class indexPerformance extends viewModelBase {
             }
             if (element.ReduceDetails) {
                 let reduceDetails: string;
-                reduceDetails = `<br/>*** Reduce details ***<br/>`;
-                reduceDetails += `Compressed leafs: ${element.ReduceDetails.NumberOfCompressedLeafs.toLocaleString()}<br/>`;
-                reduceDetails += `Modified branches: ${element.ReduceDetails.NumberOfModifiedBranches.toLocaleString()}<br/>`;
-                reduceDetails += `Modified leafs: ${element.ReduceDetails.NumberOfModifiedLeafs.toLocaleString()}`;
+
+                if (element.ReduceDetails.TreesReduceDetails) {
```

Comment:
Let's add it to @ml054 TODO list and merge it. Thoughts?

Reply:
Won't this break the studio for the meantime?

Reply:
The studio works correctly. The changes I made in this PR are:
- more details in the tooltips
- added additional timing of Linq / Jint processing (same color like in map so it doesn't break anything)